### DISCUSSION TITLE
VTherm with AC and heaters

### DISCUSSION THREAD

**Discussion by lordcoxis** [jmcollin78/versatile_thermostat#725] [Ideas]
Hi,

First I would like to thank @jmcollin78 for this amazing project. I have a large room with 3 over-climate heaters and 1 AC. Sometimes I need to turn the AC fan for a while in order for the heat generated by the heaters to be distributed to the room. To do this I created an automation that detects if the aggregate power consumption from the heaters is above X kWh for a certain time period (my main concern here is excessive power usage), and if so I turn the AC in fan-only mode for 10 minutes, which helps raise the temperature and decrease the heaters power consumption.

Would one day be possible to allow specifying AC mode per underlying entity instead of per VTherm?

Thanks

**Comment by jmcollin78** [comment]
Hello, surely not because all underlying of a VTherm should be similar. Attributes of algorithm are defined at VTherm level.

**Reply by lordcoxis** [reply]
Thanks for the explanation.
